src/library/scala/collection/immutable/Set.scala

```diff
@@ -106,6 +106,29 @@ object Set extends ImmutableSetFactory[Set] {
   }
   private[collection] def emptyInstance: Set[Any] = EmptySet
 
+  @SerialVersionUID(3L)
```

Comment:
This patch comes from https://github.com/scala/scala/commit/3e223d1c7fff68aed51820539085b908c32f6ebf#diff-e4f24e9fd33bb1494a43fee25d3b712fR130